app/src/main/res/drawable/ic_star_white_24dp.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <vector android:alpha="1.00" android:height="24dp"
-    android:viewportHeight="24.0" android:viewportWidth="24.0"
+    android:viewportHeight="22.0" android:viewportWidth="22.0"
```

Comment:
Why the change here? If some tweak to the dimensions is needed shouldn't it be done at the point of usage instead of at the icon definition?

Reply:
The actual vector drawable is only 22x22 which means that setting the viewport to 24x24 adds a hardcoded padding of 2. This became apparent when showing the stars side-by-side where the space between the stars was too much. I didn't want to add a negative margin and instead changed the vector drawable.